### ISSUE TITLE
How to train the gnn_parse on pascal person part dataset?

### ISSUE THREAD

**Issue by Anikily** [hlzhu09/Hierarchical-Human-Parsing#6]
do i need to switch the loss of noatt  to att?

Are there any other steps that need to be done?

**Comment by Anikily** [comment]
And learning rate is not specified in the paper.
